### PR TITLE
Init_project

### PR DESCRIPTION
**Pull request by dangph-1996**
==> Tittle: init project

==> Redmine: https://edu-redmine.sun-asterisk.vn/issues/16346

### CODE REVIEW

**Comment by dangph-1996** [comment]
ready

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/*
+ *= require_tree .
```
Comment (vanvtt-0952) [inline app/assets/stylesheets/application.css]: Thay vì require tree .
Chú dùng cái require file hay folder nhé

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>FresherRuby2019FootballBet</title>
```
Comment (vanvtt-0952) [inline app/views/layouts/application.html.erb]: I18n phát lun cậu

**Comment by dangph-1996** [comment]
ready

**Pull request merged**